core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/IndexFamily.java

```diff
@@ -74,6 +75,7 @@
   private PlatformType platformType;
   @Id
   private Long indexFamilyId;
+  private boolean fake;
```

Comment:
I think `fakeSequences` would be more accurate here, as the family itself isn't fake. What do y'all think?

Reply:
Yeah, probably clearer.

Reply:
That sounds a lot better. I'll change it to that.